spec/octokit/client/apps_spec.rb

```diff
@@ -16,7 +16,7 @@
     it "returns installations for an integration" do
       installations = @jwt_client.find_integration_installations
       expect(installations).to be_kind_of Array
-      assert_requested :get, github_url("/integration/installations")
```

Comment:
Done - I've added specs for the aliased methods, so we'll know if anyone removes them. (It doesn't affect this line, because both methods will now hit `/app/installations`, which I think is the right behaviour.)